Casks/t/ti-smartview-ce-for-the-ti-84-plus-family.rb

```diff
@@ -1,23 +1,22 @@
 cask "ti-smartview-ce-for-the-ti-84-plus-family" do
-  version "5.6.0.2092,0D7B013A0B0846408659344F4C28BBA8"
-  sha256 "15ba387d524161a21b8e5ef1060b495a563e2a5594e64737c0f102851f786088"
+  version "6.0.1.344"
+  sha256 "c9f86050e40d0c50b3e067e11032770025de672c767356d7bdbd090fd5610164"
 
-  url "https://education.ti.com/download/en/ed-tech/BE8220257AA241148986628D6EE332E5/#{version.csv.second}/TI-SmartView-CE-84-#{version.csv.first}.dmg"
+  url "https://d13g8g4x8s9rc1.cloudfront.net/downloads/files/cdn/smartview/84ce/#{version.major_minor_patch}/TI-SmartView-CE-84-#{version}.dmg",
+      verified: "d13g8g4x8s9rc1.cloudfront.net/downloads/files/cdn/smartview/84ce/"
   name "TI SmartView CE Emulator Software for the TI-84 Plus Family"
   desc "Software to emulate the TI 84 Plus family of calculators"
-  homepage "https://education.ti.com/en/software/details/en/BE8220257AA241148986628D6EE332E5/ti-smartview-ce-for-ti-84-plus-family"
+  homepage "https://education.ti.com/en-au/products/computer-software/ti-smartview-ce-for-84"
```

Comment:
Use the main `en` site instead of `en-au`?

```suggestion
  homepage "https://education.ti.com/en/products/computer-software/ti-smartview-ce-for-84"
```

Reply:
Thanks, but Australia is the center of **my** universe 😉

Reply:
💥